client/app/billing/autoRenew/bulk/bulk.html

```diff
@@ -0,0 +1,65 @@
+<oui-back-button data-on-click="$ctrl.goBack()">
+    <span data-ng-transclude="title"></span>
+</oui-back-button>
+<oui-message
+    class="mb-4"
+    data-type="info"
+    data-dismissable>
+    <div data-ng-transclude="infoMessage"></div>
+</oui-message>
+
+<div>
+    <strong
+        data-translate="billing_autorenew_bulk"
+        data-translate-values="{
+            count: $ctrl.servicesCount
+        }">
+    </strong>
+    <p
+        data-translate="{{:: $ctrl.description }}"
+        data-translate-values="{
+            count: $ctrl.unavailableServicesCount
+        }">
+
+    </p>
+    <ul class="mb-4">
+        <li data-ng-repeat="(reason, services) in $ctrl.unavailableServices track by reason">
+            <p>
+                <span data-translate="{{ 'billing_autorenew_bulk_unavailable_' + reason }}"></span>
+                <strong
+                    data-ng-pluralize
+                    data-count="services.length"
+                    data-when="{
+                        '1': ('billing_autorenew_bulk_service' | translate),
+                        'other': ('billing_autorenew_bulk_services' | translate:{ count: services.length })
+                    }"
+                >
+                </strong>
+                <button
+                    type="button"
+                    class="oui-link"
+                    data-ng-click="$ctrl.expand[reason] = ! $ctrl.expand[reason]"
+                    data-ng-bind="($ctrl.expand[reason] ? 'billing_autorenew_bulk_less' : 'billing_autorenew_bulk_more') | translate"></button>
```

Comment:
What do you think about adding a little bit of context of your `View more` link (#a11y concern)?

Reply:
@antleblanc As seen together the sentence beforehand and the title adds context